bind-operator/src/bind.py

```diff
@@ -346,12 +354,13 @@ def _generate_named_conf_local(
             zone_transfer_ips="",
         )
         if topology is not None:
+            transfer_list = topology.standby_units_ip.extend(secondary_transfer_ips or [])
```

Comment:
It's probably not doing any real harm here but I don't like that you're modifying the topology. I would probably
```suggestion
            transfer_list = topology.standby_units_ip + (secondary_transfer_ips or [])
```